hydradx/model/system.py

```diff
@@ -123,6 +123,6 @@ def agenthub(params, substep, state_history, prev_state, policy_input):
 
 
 def posthub(params, substep, state_history, prev_state, policy_input):
-    if 'T' in prev_state['AMM'] and prev_state['AMM']['T'] is not None:
```

Comment:
I think we can just delete these 2 rows that you commented out

Reply:
good, that's what I thought...